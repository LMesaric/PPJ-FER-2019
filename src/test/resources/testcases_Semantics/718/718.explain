semantic: not declared
note: 'prin' on line 21
