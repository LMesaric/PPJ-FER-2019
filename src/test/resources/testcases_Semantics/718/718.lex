KR_INT 15 int
IDN 15 printf
L_ZAGRADA 15 (
KR_CONST 15 const
KR_CHAR 15 char
IDN 15 format
L_UGL_ZAGRADA 15 [
D_UGL_ZAGRADA 15 ]
D_ZAGRADA 15 )
L_VIT_ZAGRADA 15 {
KR_RETURN 17 return
BROJ 17 0
TOCKAZAREZ 17 ;
D_VIT_ZAGRADA 18 }
KR_INT 20 int
IDN 20 main
L_ZAGRADA 20 (
KR_VOID 20 void
D_ZAGRADA 20 )
L_VIT_ZAGRADA 20 {
KR_RETURN 21 return
IDN 21 prin
L_ZAGRADA 21 (
NIZ_ZNAKOVA 21 "hello world!\n"
D_ZAGRADA 21 )
TOCKAZAREZ 21 ;
D_VIT_ZAGRADA 22 }
