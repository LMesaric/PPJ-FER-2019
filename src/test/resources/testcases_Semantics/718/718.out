<primarni_izraz> ::= IDN(21,prin)
